design/005-Wire-Encoding-Abstraction.md

```diff
@@ -0,0 +1,118 @@
+# Proposal: Abstraction of Wire Messages Encoding
+
+* Author(s): Sebastian Stammler (@sebastianst)
+* Affected repository: go-perun
```

Comment:
* Related issue: [go-perun#167](https://github.com/hyperledger-labs/go-perun/issues/167) 

Can you add this line as well. It helps us trace back to the implementation issues at later point in time.
Currently, I think [go-perun#167](https://github.com/hyperledger-labs/go-perun/issues/167) would be the only related issue. Other ones can be added later.